src/main/java/com/atlassian/jira/cloud/jenkins/buildinfo/pipeline/JiraSendBuildInfoStep.java

```diff
@@ -100,9 +106,16 @@ public JiraSendBuildInfoStepExecution(
         @Override
         protected JiraSendInfoResponse run() throws Exception {
             final TaskListener taskListener = getContext().get(TaskListener.class);
-            final WorkflowRun build = getContext().get(WorkflowRun.class);
+            final WorkflowRun workflowRun = getContext().get(WorkflowRun.class);
+            final Optional<String> site = pickDefaultSite(step.getSite());
```

Comment:
I'm not sure if we should do the default site lookup in the *Step. I would move it to the *Sender class where we validate other configuration params

Reply:
Okay sure, will move it to the sender class.